configuration/items.md

```diff
@@ -184,11 +186,10 @@ Examples:
 
 [Group](#groups) is a special Item type that may be used to nest or combine Items.
 Users are encouraged to apply the style guide above to group names as well as Item names.
-Two schemes are recommended for Group names:
+Two naming schemes are established in the community for Group names:
```

Comment:
As there are two contrary schemes, I thought it's good to say that they are established rather then both officially recommended.